LandSandBoat/server
# Pr/fix mother globe and slave globe behaviour mechanics to be more retail accurate

**_I affirm:_**

## What does this pull request do?

Makes the Slave Glove spawning mechanics and behaviour a little more retail. I will admit that this is not perfect because of the crappiness of "onMobRoam" timing  / mechanics. The slave globes tend to surge instead of follow closely, but it's mostly pretty good.

This is probably "best" until there is a "multiple pet" mechanic.

## Steps to test these changes

1. Start Server
2. Visit MG
3. Watch, observe, chuckle, maybe frown a little at the surging, but be mostly happy with this.
